portfolio/io.go

```diff
@@ -210,33 +211,33 @@ func parseAction(data string, tx *Tx) error {
 }
 
 func parseShares(data string, tx *Tx) error {
-	shares, err := strconv.ParseUint(data, 10, 32)
+	shares, err := decimal.NewFromString(data)
 	if err != nil {
 		return fmt.Errorf("Error parsing # shares: %v", err)
 	}
-	tx.Shares = uint32(shares)
+	tx.Shares = decimal_value.Decimal{Decimal: shares}
```

Comment:
Bit of a nit, but I feel like rather than building raw from struct args, having a `func New(dec decimal.Decimal) Decimal` would be better as an abstraction.

Reply:
Agreed, added